packages/graphql-server-express/src/apolloServerHttp.test.ts

```diff
@@ -154,7 +154,9 @@ const version = 'modern';
 describe(`GraphQL-HTTP (apolloServer) tests for ${version} express`, () => {
   describe('POST functionality', () => {
 
-    it('allows gzipped POST bodies', async () => {
+    it('allows gzipped POST bodies', async function () {
```

Comment:
Node 4 is much slower then Node 6.
I increase those 2 tests timeout since there is timeout after exhausting node abit..